Jenkinsfile

```diff
@@ -18,7 +18,7 @@ def apps = [
   [constantPrefix: "MANUALS_PUBLISHER", app: "manuals-publisher", name: "Manuals Publisher"],
   [constantPrefix: "PUBLISHER", app: "publisher", name: "Publisher"],
   [constantPrefix: "PUBLISHING_API", app: "publishing-api", name: "Publishing API"],
-  [constantPrefix: "ROUTER", app: "router", name: "Router", defaultCommitish: "master"],
```

Comment:
I'd mention in the commit message about Router now having a `deployed-to-production` tag set up.